integration/testrunner/grpc.go

```diff
@@ -145,14 +161,24 @@ func testGRPCEndpoint(ctx context.Context, t *testing.T, g *GRPC) {
 		} else {
 			t.Errorf("Did not get err as expected. Instead got reply: %v", reply)
 		}
-
-		query := g.makeQueryForGRPCUnary(id)
-		queryIfAuditLogExistsWithRetries(ctx, t, query, g.Config, "unary_fail")
+		query := g.makeQueryForGRPC(id)
+		t.Log(query.Q)
+		results := queryIfAuditLogsExists(ctx, t, query, g.Config, "unary_fail")
+		wantNum := 1
+		if len(results) < wantNum {
```

Comment:
Should it be != instead of <

Reply:
There are some inconsistency behavior for this test. Sometimes the log number is 1, and sometimes it's 2. I am thinking we fix that bug in another PR.
Context: https://github.com/abcxyz/lumberjack/actions/runs/4130970764/jobs/7138241725 and https://github.com/abcxyz/lumberjack/actions/runs/4130843487/jobs/7138014374

Reply:
fixed in commit: c9ddda9